docs/installation/manual/configuring_drupal.md

```diff
@@ -0,0 +1,163 @@
+
+Take note of some of the other comments in the below bash script for an idea of what the other components are expected, and which may be considered optional.
+
+```bash
+cd /opt/drupal
+# This is a convenience piece that will help speed up most of the rest of our
+# process working with Composer and Drupal.
+sudo -u www-data composer require zaporylie/composer-drupal-optimizations:^1.0
+# Since islandora_defaults is near the bottom of the dependency chain, requiring
+# it will get most of the modules and libraries we need to deploy a standard
+# Islandora site.
+sudo -u www-data composer require islandora/islandora_defaults:dev-8.x-1.x
+# These can be considered important or required depending on your site's
+# requirements; some of them represent dependencies of Islandora submodules.
+sudo -u www-data composer require drupal/pdf:1.x-dev
+sudo -u www-data composer require drupal/rest_oai_pmh:^1.0
+sudo -u www-data composer require drupal/facets:^1.3
+sudo -u www-data composer require drupal/restui:^1.16
+sudo -u www-data composer require drupal/rdfui:^1.0-beta1
+sudo -u www-data composer require drupal/content_browser:^1.0@alpha
+# These tend to be good to enable for a development environment, or just for a
+# higher quality of life when managing Islandora. That being said, devel should
+# NEVER be enabled on a production environment, as it intentionally gives the
+# user tools that compromise the security of a site.
+sudo -u www-data composer require drupal/console:~1.0
+sudo -u www-data composer require drupal/devel:^2.0
+sudo -u www-data composer require drupal/admin_toolbar:^2.0
+# Islandora also provides a theme called Carapace designed to work well out of
+# the box with an Islandora site.
+sudo -u www-data composer require islandora/carapace:dev-8.x-3.x
+```
+
+### Enabling Downloaded Components
+
+Components we've now downloaded using `composer require` can be enabled simultaneously via `drush`, which will ensure they are installed in the correct dependent order. Enabling `islandora_defaults` will also ensure all content types and configurations are set up in Islandora. The installation process for all of these modules will likely take some time.
+
+!!! notice
+    This list of modules assumes that all of the above components were downloaded using `composer require`; if this is not the case, you may need to pare down this list manually. It also includes `devel`, which again, should not be enabled on production sites.
+
+```bash
+cd /opt/drupal
+drush -y en rdf responsive_image devel syslog serialization basic_auth rest restui search_api_solr search_api_solr_defaults facets content_browser pdf admin_toolbar islandora_defaults controlled_access_terms_defaults islandora_breadcrumbs islandora_iiif islandora_oaipmh
```

Comment:
You may hit the following error when running this command if the islandora_core_feature module hasn't been enabled yet:
```
Error: Call to a member function getSettings() on null in /opt/drupal/web/modules/contrib/islandora/modules/islandora_text_extraction/islandora_text_extraction.install on line 16 #0 [internal function]: islandora_text_extraction_install()
#1 /opt/drupal/web/core/lib/Drupal/Core/Extension/ModuleHandler.php(392): call_user_func_array('islandora_text_...', Array)
#2 /opt/drupal/web/core/lib/Drupal/Core/Extension/ModuleInstaller.php(323): Drupal\Core\Extension\ModuleHandler->invoke('islandora_text_...', 'install')
```
The enable command doesn't seem to enable modules in a specific order, so it might be best to add an instruction to enable islandora_core_feature first, and then the other modules.

Reply:
Actually, enabling islandora_core_feature before-hand doesn't resolve this.... maybe this is a Drupal 8.8 thing....

Reply:
I also had to run this twice.  Although the offending file itself I think can just be deleted: https://github.com/Islandora/islandora/blob/8.x-1.x/modules/islandora_text_extraction/islandora_text_extraction.install

.txt is added by default.  Feels crufty.

Reply:
https://github.com/Islandora/islandora/pull/748